ballerina/http_client_connection_pool.bal

```diff
@@ -27,11 +27,13 @@ configurable int maxActiveStreamsPerConnection = 100;
 # + maxIdleConnections - Maximum number of idle connections allowed per pool.
 # + waitTime - Maximum amount of time (in seconds), the client should wait for an idle connection before it sends an error when the pool is exhausted
 # + maxActiveStreamsPerConnection - Maximum active streams per connection. This only applies to HTTP/2. Default value is 100
+# + evictableIdleTime - Evictable time for an idle connection in seconds. Default value is 5 minutes
 public type PoolConfiguration record {|
```

Comment:
There is another configuration for the eviction interval right? - `timeBetweenEvictionRuns`
The default value for this configuration is 30 seconds. Shouldn't we expose that as well?
@shafreenAnfar WDYT?

Reply:
Updated